docs/concepts/multiaddr.md

```diff
@@ -0,0 +1,40 @@
+---
+title: 'Multi-addresses'
+description: Multi-addresses and how they are used in IPFS
+---
+
+# Understanding Multi-Addresses in IPFS
+
+A [multiaddr](https://github.com/multiformats/multiaddr) encodes multiple layers of addressing information into a single "future-proof" path structure.
```

Comment:
Suggestion (with a question):

Since this is the first time "multiaddr" is used on the page, would it make sense to use the term "Multi-address" since that's used in the tile of the doc? Most readers will probably infer that multiaddr is short for multi-address, but I don't think we should assume.  So it would read
```
A [multi-address](https://github.com/multiformats/multiaddr) (multiaddr) encodes multiple layers...
```

Also, is it "multiaddress" or "multi-address"? I'm unsure from https://github.com/multiformats/multiaddr